backend/python/coqui/coqui_server.py

```diff
@@ -45,8 +45,10 @@ def LoadModel(self, request, context):
             return backend_pb2.Result(success=False, message="CUDA is not available")
 
         self.AudioPath = None
+
         # List available 🐸TTS models
-        print(TTS().list_models())
+        print(TTS().list_models().list_models())
```

Comment:
this looks like a leftover, or is it wanted?

Reply:
I was not sure, I will remove it then. 

I am planning to include an endpoint to list models/speakers in this PR.